server/routes/updateAddress.ts

```diff
@@ -0,0 +1,59 @@
+import { Request, Response, NextFunction } from 'express';
+import { factory, formatFilename } from '../../shared/logging';
+import { DB, sequelize } from '../database';
+import { Errors } from "./verifyAddress";
+
+const log = factory.getLogger(formatFilename(__filename));
+
+// update ghost address for imported discourse users by new address
+const updateAddress = async (models: DB, req: Request, res: Response, next: NextFunction) => {
+    const { address, chain, userId } = req.body
+    if (!address) {
+        return next(new Error(Errors.NoAddress));
+    }
+    if (!chain) {
+        return next(new Error(Errors.NoChain));
+    }
+    if (!userId) {
+        return next(new Error(Errors.NoUserId));
+    }
+
+    const { id: ghostAddressId } = await models.Address.scope('withPrivateData').findOne({
+        where: { chain: 'injective', ghost_address: true, user_id: userId }
```

Comment:
Should use chain parameter from body rather than hardcoded, here and below.

Reply:
sorry forgot to remove it. It was for the testing. Good Catch !